examples/coverage/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  stdenv,
+  lib,
+  hax,
+  coqPackages,
+  gnused,
+  craneLib,
+  bat,
+  coqGeneratedCore ? import ../../proof-libs/coq/coq {inherit stdenv coqPackages;},
+}:
+let
+  commonArgs = import ../commonArgs.nix {inherit craneLib lib;};
+  cargoArtifacts = craneLib.buildDepsOnly commonArgs;
+in
+  craneLib.mkCargoDerivation (commonArgs
+    // {
+      inherit cargoArtifacts;
+      pname = "coverage";
+      doCheck = false;
+      buildPhaseCargoCommand = ''
+        cd examples/coverage
+        cargo hax into coq
+
+        cd proofs/coq/extraction
```

Comment:
Does the `cd` issue originate from here? Does it fail to generate something when running `cargo hax` here?

Reply:
Seemed to be an issue with having to do `cd ../../../../` to get back to the start directory after running the script.